informationswissenschaft-theorie-methode-praxis.csl

```diff
@@ -0,0 +1,552 @@
+        <date variable="issued" form="numeric" date-parts="year-month-day"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <choose>
+      <if variable="locator">
+        <text macro="locator"/>
+      </if>
+      <else>
+        <group delimiter="&#160;">
+          <label variable="page" form="short"/>
+          <text variable="page"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="locator">
+    <group delimiter="&#160;">
+      <label variable="locator" form="short"/>
+      <text variable="locator"/>
+    </group>
+  </macro>
+  <macro name="book-series">
+    <choose>
+      <if type="book chapter paper-conference entry-dictionary entry-encyclopedia" match="any">
+        <group prefix="(" suffix=")" delimiter=" ">
+          <text variable="collection-title"/>
+          <choose>
+            <if is-numeric="collection-number">
+              <number variable="collection-number"/>
+            </if>
+          </choose>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="newspaper-edition">
+    <choose>
+      <if type="article-newspaper">
+        <choose>
+          <if is-numeric="edition">
+            <group delimiter="&#160;">
+              <number variable="edition" form="ordinal"/>
+              <label variable="edition"/>
+            </group>
+          </if>
+          <else>
+            <text variable="edition"/>
+          </else>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="artwork-description">
+    <choose>
+      <if type="graphic">
+        <group delimiter=", ">
+          <text variable="medium"/>
+          <text variable="dimensions"/>
+          <text variable="genre"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="archive-location">
+    <choose>
+      <if variable="archive">
+        <group delimiter=", ">
+          <text variable="archive"/>
+          <text variable="archive_location"/>
+          <text variable="call-number" prefix="Signatur: "/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="url-web-documents-only">
+    <choose>
+      <if type="webpage post post-weblog dataset" match="any">
+        <text macro="url-or-doi"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="url-non-web-documents">
+    <choose>
+      <if type="webpage post post-weblog dataset" match="none">
+        <choose>
+          <if variable="DOI URL" match="any">
+            <group delimiter=": ">
+              <text term="online" text-case="capitalize-first"/>
+              <text macro="url-or-doi"/>
+            </group>
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="url-or-doi">
+    <choose>
+      <if variable="DOI">
+        <group delimiter=", ">
+          <group delimiter="">
+            <text value="&lt;"/>
+            <text variable="DOI" prefix="https://doi.org/"/>
+            <text value="&gt;"/>
```

Comment:
Can you move the < and > text values to be part of the prefix and suffix on the DOI? Then you can remove the group here

Reply:
Sure, sorry about that!